crates/core/tedge_agent/src/entity_manager/server.rs

```diff
@@ -24,6 +24,7 @@ use tracing::error;
 pub enum EntityStoreRequest {
     Get(EntityTopicId),
     Create(EntityRegistrationMessage),
+    Update(EntityTopicId, EntityTopicId),
```

Comment:
`Update` gives little clue on what is done by this request. This is even confusing as this actually to update the parent of an entity.

Reply:
Since the scope of update expanded beyond updating the parent (can update health endpoint as well), I've left it as `Update` itself with `UpdateEntityMessage` as  a parameter.